.github/workflows/codeql.yml

```
@@ -0,0 +1,57 @@
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - language: java-kotlin
+          build-mode: autobuild
+
+    steps:
+    - name: Checkout repository
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)